extensions/sql-bindings/src/common/utils.ts

```diff
@@ -162,6 +162,28 @@ export function escapeClosingBrackets(str: string): string {
 	return str.replace(']', ']]');
 }
 
+/**
+ * Gets a unique object name by removing all special characters
+ * @param objectName can include brackets/periods and user entered special characters
+ * @returns the object name without any special characters
+ */
+export function getUniqueObjectName(objectName: string): string {
+	return objectName.replace(/[^a-zA-Z0-9 ]/g, '');
+}
+
+/**
+ * Check to see if the input from user entered is valid
+ * @param input from user input
+ * @returns false if the input is empty or has special characters, true if the input is valid
+ */
+export function checkInput(input: string): boolean {
```

Comment:
This is a pretty generic name for the function, I'd suggest something clearer like `hasOnlyAlphanumericCharacters`.

But also I think it would be better to have this return `string | undefined` and then return the error message associated with the specific error. So have two errors, one for if it's empty and another if it has non-alphanumeric characters. Then a better name would be `validateFunctionName` I think

Reply:
With new fix user will see this:
- Special Characters
<img width="520" alt="Screen Shot 2022-04-07 at 7 38 08 PM" src="https://user-images.githubusercontent.com/23587151/162352485-60a1c56d-291b-48a3-9ed4-e46d962893eb.png">

- Empty
<img width="530" alt="Screen Shot 2022-04-07 at 7 38 12 PM" src="https://user-images.githubusercontent.com/23587151/162352495-1f5dd575-9cf2-4e88-bc3a-72918940707c.png">